### PR TITLE
Add openstack cluster: openstack-vh-mecha

### PR DESCRIPTION
**Pull request by pierreprinetti**
This is the team-managed cluster running on a baremetal VM that we
intend to use for presubmits.

### CODE REVIEW

File: clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/boskos.jsonnet [hunk 1]
```diff
@@ -70,4 +70,5 @@ dashboard.new(
 .addPanel(statePanel(iaas="openstack", displayName="OpenStack"), gridPos={h: 9, w: 24, x: 0, y: 0})
 .addPanel(statePanel(iaas="openstack-kuryr", displayName="OpenStack Kuryr"), gridPos={h: 9, w: 24, x: 0, y: 0})
 .addPanel(statePanel(iaas="openstack-vexxhost", displayName="OpenStack Vexxhost"), gridPos={h: 9, w: 24, x: 0, y: 0})
+.addPanel(statePanel(iaas="openstack-vh-central", displayName="OpenStack Vexxhost"), gridPos={h: 9, w: 24, x: 0, y: 0})
```
Comment (EmilienM) [inline clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/boskos.jsonnet:73]: let's use another Display name

**Comment by pierreprinetti** [comment]
Tests look OK.

/assign @hongkailiu
like the bot suggests

**Comment by pierreprinetti** [comment]
I have changed the name to something more easy to remember.

@EmilienM can you please PTAL again?

**Comment by pierreprinetti** [comment]
/retest

The secret is now in the vault.

**Comment by pierreprinetti** [comment]
/retest

The secret is _now_ in the vault

**Comment by pierreprinetti** [comment]
/assign petr-muller

Is everything alright?

**Comment by petr-muller** [comment]
/unassign

Sorry, I cannot take more ad-hoc work now

**Comment by hongkailiu** [comment]
I cannot say about the changes on the step.
The dashboard part and the boskos config look good to me.

/approve

**Pull request merged**